modules/api-support-tiers.adoc

```diff
@@ -0,0 +1,130 @@
+
+Components and developer tools that receive continuous updates through the Operator Hub, referred to as Operators and operands, should be considered API tier 3.  Developers should use caution and understand how these components may change with each release. Users are encouraged to consult the compatibility guidelines documented by the component.
+
+[discrete]
+[id="api-tier-4_{context}"]
+== API tier 4
+No compatibility is provided. API and AOE can change at any point. These capabilities should not be used by applications needing long-term support.
+
+It is common practice for Operators to use custom resource definitions (CRDs) internally to accomplish a task.  These objects are not meant for use by actors external to the Operator and are intended to be hidden.  If any CRD is not meant for use by actors external to the Operator, the `operators.operatorframework.io/internal-objects` annotation in the Operators `ClusterServiceVersion` (CSV) must be specified, and that signals that the corresponding resource is internal use only.
+
+[id="mapping-support-tiers-to-api-groups_{context}"]
+= Mapping API tiers to API groups
+
+For each API tier defined by Red Hat, we provide a mapping table for specific API groups where the upstream communities are committed to maintain forward compatibility. Any API group that does not specify an explicit compatibility level is assigned API tier 3 by default.
+
+[id="mapping-support-tiers-to-kubernetes-api-groups_{context}"]
+== Support for Kubernetes API groups
+
+API groups that end with the suffix `*.k8s.io `or have the form `version.<name>` with no suffix are governed by the Kubernetes deprecation policy and follow a general mapping between API version exposed and corresponding support tier unless otherwise specified.
+
+[cols="2",options="header"]
+|===
+|API version example
+|API tier
+
+|`v1`
+|Tier 1
+
+|`v1beta1`
+|Tier 2
+
+|`v1alpha1`
+|Tier 4
+
+|===
+
+[id="mapping-support-tiers-to-openshift-api-groups_{context}"]
+== Support for OpenShift API groups
+
+API groups that end with the suffix `*.openshift.io` are governed by the {product-title} deprecation policy and follow a general mapping between API version exposed and corresponding compatibility level unless otherwise specified.
+
+[cols="2",options="header"]
+|===
+|API version example
+|API tier
+
+|`v1`
+|Tier 1
+
+|`v1beta1`
+|Tier 2
+
+|`v1alpha1`
+|If installed by default, tier 4.
+
+|===
+
+[id="mapping-support-tiers-to-monitoring-api-groups_{context}"]
+== Support for Monitoring API groups
+
+API groups that end with the suffix `monitoring.coreos.com` have the following mapping:
+
+[cols="2",options="header"]
+|===
+|API version example
+|API tier
+
+|`v1`
+|Tier 1
+
+|===
+
+[id="mapping-support-tiers-to-coreos-api-groups_{context}"]
+== Support for CoreOS API groups
```

Comment:
@bparees @ecordell in the interest of publishing and then evolving , we can remove the coreos and metal3 section initially and then come back and add different groups for operator-framework if needed.